### DISCUSSION TITLE
Adjust UI in visionOS

### DISCUSSION THREAD

**Discussion by ianmagallan** [kingslay/KSPlayer#751] [Ideas]
Hi 👋 I am writing this post to understand whether or not you'd find interesting to adjust the visionOS UI for KSPlayer. So far, I've forked KSPlayer and in my own fork I've already changed it, and it looks like so:

<img src="https://github.com/kingslay/KSPlayer/assets/24428301/1b866c73-6c71-460c-b77c-fa1b70207d20" width="650"/>
<img src="https://github.com/kingslay/KSPlayer/assets/24428301/df8a3476-7563-484a-8db3-ca5a74195a0a" width="650"/>
<img src="https://github.com/kingslay/KSPlayer/assets/24428301/419283bb-17c4-4e4e-b0ff-f2a097392f1c" width="650"/>

Additionally, the volume bar has a slider and can be lowered gradually.

Let me know if you'd be interested on having this in the project.

@kingslay If you can remember this, we've been exchanging emails about this topic the past days, and I decided to open a discussion here as per the readme.me file of the project, to make sure that there is need/interest for it :)

**Comment by kingslay** [comment]
我觉得你这个UI很好 你可以发个pull request 合并进来

**Reply by ianmagallan** [reply]
This PR has been already merged to main https://github.com/kingslay/KSPlayer/pull/753
